<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inventoryEditorForm</class>
 <widget class="QWidget" name="inventoryEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="inventoryEditorLayout">
     <item>
      <widget class="QLabel" name="deviceEditorTitle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Edit Device</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="inventoryFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="deviceLabel">
         <property name="text">
          <string>Device</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="availableLabel">
         <property name="text">
          <string>Available</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="avialableCheckbox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="motLabel">
         <property name="text">
          <string>Next MOT</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="categoryLabel">
         <property name="text">
          <string>Category</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="categoryEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="infoLabel">
         <property name="text">
          <string>Info</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPlainTextEdit" name="infoEdit"/>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="nextMotEdit" stretch="1,3">
         <item>
          <widget class="QComboBox" name="monthCombo">
           <item>
            <property name="text">
             <string>Jan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Feb</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Apr</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>May</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jun</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jul</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Aug</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sep</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Oct</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nov</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dec</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="yearSpinner"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="editDeviceButtonLayout">
       <item>
        <widget class="QPushButton" name="cancelDeviceEditButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="confirmDeviceEditButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
